lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -1,36 +1,45 @@
 # Guidelines for AWS modules
 
-## Getting Started
+The Ansible AWS modules and these guidelines are maintained by the Ansible AWS Working Group.  For
+further information see 
+[the AWS working group community page](https://github.com/ansible/community/tree/master/group-aws).  
+If you are planning to contribute AWS modules to Ansible then getting in touch with the the working
+group will be a good way to start, especially because a similar module may already be under
+development.
 
-Since Ansible 2.0, it is required that all new AWS modules are written to use boto3.
+## Choose Boto3 not Boto
 
-Prior to 2.0, modules may have been written in boto or boto3. The effort to port all modules to boto3 has begun.
+Starting from Ansible 2.0, it has been required that all new AWS modules are written to use boto3.
+Please do not add new dependencies to the old boto library.
+
+Prior to 2.0, modules may have been written in boto or boto3. The effort to port all modules to
+boto3 has begun.  From Ansible 2.4 it is permissible for modules which previously required boto to
+start to require boto3 in order to deliver the functionality previously supported with boto and the
+boto dependency can be deleted.
 
 ## Bug fixing
 
-Bug fixes to code that relies on boto will still be accepted. When possible, the code should be ported to use boto3. 
+Bug fixes to code that relies on boto will still be accepted. When possible, the code should be
+ported to use boto3.
 
 ## Naming your module
 
-Base the name of the module on the part of AWS that
-you actually use. (A good rule of thumb is to take
-whatever module you use with boto as a starting point).
+Base the name of the module on the part of AWS that you actually use. (A good rule of thumb is to
+take whatever module you use with boto as a starting point).  Don't further abbreviate names - if
+something is a well known abbreviation due to it being a major component of AWS, that's fine, but
+don't create new ones independently (e.g. VPC, ELB, etc. are fine).
 
-Don't further abbreviate names - if something is a well
-known abbreviation due to it being a major component of
-AWS, that's fine, but don't create new ones independently
-(e.g. VPC, ELB, etc. are fine)
+Unless the name of your service is quite unique, please consider using "aws_" as a prefix.  For
+example "aws_lambda".
 
 ## Adding new features
 
-Try to keep backward compatibility with relatively recent
-versions of boto3. That means that if you want to implement some
-functionality that uses a new feature of boto3, it should only
-fail if that feature actually needs to be run, with a message
-saying which version of boto3 is needed.
+Try to keep backward compatibility with relatively recent versions of boto3. That means that if you
+want to implement some functionality that uses a new feature of boto3, it should only fail if that
+feature actually needs to be run, with a message saying which version of boto3 is needed.
```

Comment:
s/saying which version of boto3 is needed/stating the minimum required version of boto3/